inductiva/structures/scenarios/plate_linear_elastic/material_utils.py

```diff
@@ -0,0 +1,41 @@
+"""Utils to create the material."""
+
+import json
+
+
+class IsotropicLinearElasticMaterial:
+    """Linear elastic isotropic material.
+
+    Attributes:
+        young_modulus: A float representing the Young's modulus.
+        poisson_ratio: A float representing the Poisson's ratio.
```

Comment:
No need for the bits "A float representing". Add "... of the material" at the end, though.